prow/plugins/assign/assign.go

```diff
@@ -134,13 +157,39 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(unassign) > 0 {
 		log.Printf("Removing assignees from %s/%s#%d: %v", ae.org, ae.repo, ae.number, unassign)
 		if err := gc.UnassignIssue(ae.org, ae.repo, ae.number, unassign); err != nil {
-			return err
+			msg := fmt.Sprintf("Unassign failed: %v.", err.Error())
+
+			ic := github.IssueComment{
```

Comment:
This is not the right way to go. Rather than fooling `plugins.FormatResponse` into thinking we have a `github.IssueComment` when we don't, we should add a new method to `plugins`.